test/src/test/java/org/corfudb/integration/ClusterReconfigIT.java

```diff
@@ -457,6 +461,54 @@ public void test3NodeBootstrapUtility() throws Exception {
         shutdownCorfuServer(corfuServer_3);
     }
 
+    /**
+     * Setup a cluster of 3 nodes. Bootstrap one of the nodes.
+     * Bootstrap all the nodes using the BootstrapUtil. It should assert that the node already
+     * bootstrapped is with the same layout and then bootstrap the cluster.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void test3NodeBootstrapUtilityWithException() throws Exception {
```

Comment:
Can you test the case, when you bootstrap a server that has been bootstrapped with a different layout? you added a new exception path that is not tested.